crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -233,6 +233,19 @@ impl SuiObjectDataOptions {
         }
     }
 
+    /// return full sui object info
+    pub fn sui_object_default() -> Self {
+        Self {
+            show_bcs: false,
+            show_type: true,
+            show_owner: true,
+            show_previous_transaction: true,
+            show_display: false,
+            show_content: false,
+            show_storage_rebate: false,
+        }
+    }
```

Comment:
```suggestion
```

we already added the default trait on line 186. See discussion in https://github.com/MystenLabs/sui/pull/8817#discussion_r1125508513 for why all values should default to be false

Reply:
For places that use this combination of config, use `SuiObjectDataOptions::new().with_owner().with_previous_transaction().with_owner()` instead